native-schema-registry/src/main/java/com/amazonaws/services/schemaregistry/ExceptionWriter.java

```diff
@@ -30,4 +34,22 @@ private static void writeException(String message, C_GlueSchemaRegistryErrorPoin
         throwError(errorPointer, errorMessageAsCString.get(), FAILURE_CODE);
         errorMessageAsCString.close();
     }
+
+    private static String getFullStackTraceMessage(Throwable throwable){
+        if (throwable == null || throwable.getMessage() == null) {
+            return "NULL is thrown";
+        }
+        if (throwable.getStackTrace() == null){
+            return "Stacktrace is NULL";
+        }
+        StringJoiner messageJoiner = new StringJoiner("\n");
+        messageJoiner.add(throwable.getMessage());
+        for (StackTraceElement stackTraceElement : throwable.getStackTrace()) {
+            if (stackTraceElement == null) continue;
+            messageJoiner.add(stackTraceElement.toString());
```

Comment:
Can you please paste an example of how this error message looks like? Shouldn't we be adding the `message` part of the exception as well?

Reply:
We have added the message of the exception in the very beginning of the returned message. If you are referring to the stacktrace element here, it does not have message and `stackTraceElement.toString()` will give us the information we are interested in.

Reply:
I  have added an example